axum-extra/Cargo.toml

```diff
@@ -40,7 +40,7 @@ typed-routing = ["dep:axum-macros", "dep:percent-encoding", "dep:serde_html_form
 
 [dependencies]
 axum = { path = "../axum", version = "0.7.6", default-features = false }
```

Comment:
Shouldn't this be also bumped? Arguably, it doesn't matter since there are no changes except for docs, but I'd rather keep it in sync.

Reply:
Fixed.